lib/vnet/src/lib.rs

```diff
@@ -516,6 +516,9 @@ pub enum NetworkError {
     /// A call to write returned 0
     #[error("write returned 0")]
     WriteZero,
+    /// OS error
+    #[error("operating system error({0})")]
+    OsError(i32),
```

Comment:
We are our own OS, I don't see how having an upstream OS error here makes sense. 
We should map OS errors to the appropriate error variant.

Reply:
Fixed in https://github.com/wasmerio/wasmer/commit/4473be1b043cb772b2fa897835a75a6b2a8c2e56